crates/holochain/src/core/ribosome/host_fn/entry_type_properties.rs

```diff
@@ -1,13 +1,12 @@
 use crate::core::ribosome::error::RibosomeResult;
 use crate::core::ribosome::CallContext;
 use crate::core::ribosome::RibosomeT;
-use holochain_types::prelude::*;
 use std::sync::Arc;
 
 pub fn entry_type_properties(
     _ribosome: Arc<impl RibosomeT>,
     _call_context: Arc<CallContext>,
-    _input: EntryTypePropertiesInput,
-) -> RibosomeResult<EntryTypePropertiesOutput> {
+    _input: (),
```

Comment:
see above

Reply:
this is `unimplemented!()` on the host side